main/inc/lib/attendance.lib.php

```diff
@@ -1202,17 +1205,39 @@ public function get_next_attendance_calendar_datetime($attendance_id)
 	 * @param	int $attendance_id
 	 * @return	int score
 	 */
-	public function get_user_score($user_id, $attendance_id)
+	public function get_user_score($user_id, $attendance_id, $groupId = null)
 	{
 		$tbl_attendance_result 	= Database::get_course_table(TABLE_ATTENDANCE_RESULT);
+        $tbl_attendance_sheet = Database::get_course_table(TABLE_ATTENDANCE_SHEET);
+        $tbl_attendance_cal_rel_group = Database::get_course_table(TABLE_ATTENDANCE_CALENDAR_REL_GROUP);
+        $tbl_attendance_cal = Database::get_course_table(TABLE_ATTENDANCE_CALENDAR);
 		$user_id = intval($user_id);
 		$attendance_id = intval($attendance_id);
 		$course_id = api_get_course_int_id();
-		$sql = "SELECT score FROM $tbl_attendance_result
-				WHERE
-					c_id = $course_id AND
-					user_id='$user_id' AND
-					attendance_id='$attendance_id'";
+        if (empty($groupId))
+        {
```

Comment:
Control statements MUST have their opening bracket on the same line (only classes, methods and functions don't)